### ISSUE TITLE
How to get EXT-X-MAP from m3u8 object?

### ISSUE THREAD

**Issue by Cosmicoppai** [globocom/m3u8#329]

**Comment by davemevans** [comment]
Yes, the uri and byterange of the map associated with a particular segment are available. See https://github.com/globocom/m3u8/blob/master/tests/test_model.py#L825
